pygmt/tests/test_clib_put_vector.py

```diff
@@ -170,16 +170,26 @@ def test_put_vector_invalid_dtype():
     """
     Check that it fails with an exception for invalid data types.
     """
-    with clib.Session() as lib:
-        dataset = lib.create_data(
-            family="GMT_IS_DATASET|GMT_VIA_VECTOR",
-            geometry="GMT_IS_POINT",
-            mode="GMT_CONTAINER_ONLY",
-            dim=[2, 3, 1, 0],  # columns, rows, layers, dtype
-        )
-        data = np.array([37, 12, 556], dtype="object")
-        with pytest.raises(GMTInvalidInput):
-            lib.put_vector(dataset, column=1, vector=data)
+    for dtype in [
+        np.bool_,
+        np.bytes_,
+        np.complex64,
+        np.complex128,
+        np.complex256,
```

Comment:
So `np.complex256` raises an error in Windows like `AttributeError: module 'numpy' has no attribute 'complex256'`

Reply:
Ah nevermind, was too slow. Saw the change at c5ca123f3ea5b5c33a93479390a73b5260b717ef